README.md

```diff
@@ -65,7 +66,7 @@ The Orders page contains the following filters:
 
 On the 'Orders Awaiting Approval' page only orders that should be approved according to the Approval workflow, are displayed. The page displays the same table and the same filters as 'Orders' page.
 
-The 'Awaiting Approval' page is visible only to users with the role 'Approver'.
+The 'Awaiting Approval' page is visible only to users with the role 'Approver'. The role 'Approver' depends on the approval workflow and can be 'Approver 1', 'Approver 2', etc.
 
 ![Awaiting Approval Page](docs/media/screen-awaiting-approval.png)
```

Comment:
This is an outdated image of the awaiting approval page, we should update it because we have some changes there (eg: added the filter by status, renaming in column titles)